src/gebt_poc/solver.cpp

```diff
@@ -16,22 +16,46 @@ UserDefinedQuadrature::UserDefinedQuadrature(
 }
 
 void Interpolate(
```

Comment:
Should we rename this `InterpolateQuaternion`, since it can't be used for anything else?

Reply:
Both of these functions are really setup to interpolate the nodal values, how about `InterpolateNodalValues` and `InterpolateNodalValueDerivatives`?

Reply:
That works for me.